### ISSUE TITLE
ModuleNotFoundError: No module named 'ont_fast5_api.fast5_interface'

### ISSUE THREAD

**Issue by drs357** [nanoporetech/medaka#179]
Hello,
When I run medaka consensus, I get the following error:

ModuleNotFoundError: No module named 'ont_fast5_api.fast5_interface'

I am running it in conda 4.6.14
and medaka was installed using conda from the bioconda library. 

Thank you

**Comment by cvdelannoy** [comment]
just had this one too, in my case I noticed that conda doesn't install the latest version if you only specify the bioconda channel (it's stuck at 0.11.5). I solved it by adding the conda-forge and specified the latest version to be sure:
```
conda create -n medaka -c bioconda -c conda-forge medaka=1.0.3
```
With this version the ModuleNotFoundError didn't occur anymore. Still a bug I guess, but perhaps this helps in the meantime.
